src/core/Elsa.Abstractions/Models/WorkflowInstance.cs

```diff
@@ -33,13 +34,24 @@ public WorkflowInstance()
         public WorkflowInputReference? Input { get; set; }
         public WorkflowOutputReference? Output { get; set; }
         public IDictionary<string, IDictionary<string, object?>> ActivityData { get; set; } = new Dictionary<string, IDictionary<string, object?>>();
+        public IDictionary<string, object?> MetaData { get; set; } = new Dictionary<string, object?>();

```

Comment:
"Metadata" is one word, so the capitalization should be: `Metadata` (notice lowercase 'd'). This should be applied everywhere.